### DISCUSSION TITLE
Automatic installation of .dmg and .pkg with Homebrew?

### DISCUSSION THREAD

**Discussion by tddschn** [Homebrew/discussions#2121] [Casks] [answered]
So I downloaded a .dmg or .pkg file and wanted to install it the easiest way possible, I know Homebrew cask can do that with a defined Cask file, but how do I make Homebrew work with the .dmg or .pkg file without a Cask file?

**Comment by SMillerDev** [comment, marked as answer]
You can not. A cask file defines what to download, from where and where to install that. Without thus file homebrew can't deal with packages.

**Reply by tddschn** [reply]
(I'm curious) Why hasn't anyone implement a utility / convenience (official) command that does this? 
It seems that a lot of `brew install --cask`'s code can be reused?

**Reply by SMillerDev** [reply]
Probably because it's less work to install the stuff manually.

**Reply by tddschn** [reply]
Makes sense, I barely need to manually install stuff with Homebrew installed. :)
